src/force/CabanaPD_Force_PMB.hpp

```diff
@@ -87,12 +90,12 @@ class Force<ExecutionSpace, ForceModel<PMB, Elastic>>
 
     template <class ParticleType, class NeighListType, class ParallelType>
     void computeWeightedVolume( ParticleType&, const NeighListType&,
-                                const ParallelType )
+                                const ParallelType ) const
```

Comment:
@streeve What is the extra "const"? (Same in Line 98 below)

Reply:
The `const` changes below are necessary to update the timers automatically. This was just done for consistency once I noticed (technically a separate change)